### PR TITLE
Add changelog and docs for 6.0.0 release

### PR DESCRIPTION
**Pull request by ryan-lane**

### CODE REVIEW

File: docs/source/advanced/acls.markdown [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+
+```python
+acl_module_check(
+    resource_type='credential',
+    action='update',
+    resource_id=id,
+)
+```
+
+This check controls update access to specific credentials. Fine-grained controls can be applied using the provided `resource_id`. Note that if you're controlling access to this, you probably also want to control access to [Revert credential](#Revert credential).
```
Comment (skiptomyliu) [inline docs/source/advanced/acls.markdown]: I think you may need to convert the anchor link from `#Revert credential` to `#revert-credential`

File: docs/source/advanced/acls.markdown [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+
+```python
+acl_module_check(
+    resource_type='credential',
+    action='revert',
+    resource_id=id,
+)
+```
+
+This check controls revert access to specific credentials. Fine-grained controls can be applied using the provided `resource_id`. Note that if you're controlling access to this, you probably also want to control access to [Update credential](#Update credential).
```
Comment (skiptomyliu) [inline docs/source/advanced/acls.markdown]: ditto here on link format `#update-credential`

File: docs/source/advanced/acls.markdown [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+
+```python
+acl_module_check(
+    resource_type='service',
+    action='update',
+    resource_id=id,
+)
+```
+
+This check controls update access to specific services. Fine-grained controls can be applied using the provided `resource_id`. Note that if you're controlling access to this, you probably also want to control access to [Revert service](#Revert service).
```
Comment (skiptomyliu) [inline docs/source/advanced/acls.markdown]: marking this link too

File: docs/source/advanced/acls.markdown [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+
+```python
+acl_module_check(
+    resource_type='service',
+    action='revert',
+    resource_id=id,
+)
+```
+
+This check controls revert access to specific services. Fine-grained controls can be applied using the provided `resource_id`. Note that if you're controlling access to this, you probably also want to control access to [Update service](#Update service).
```
Comment (skiptomyliu) [inline docs/source/advanced/acls.markdown]: marking this link `#update-service`

**Comment by ryan-lane** [comment]
I'll followup on the links. I'm changing the doc site over to sphinx right now, because I can't figure out how to get the middleman build to work, since it's an ancient version.

**Pull request merged**